### PR TITLE
Config (.ini) file

### LINKED ISSUE #15: Introduce a config file for keybindings, and other features
There is a recent pull request https://github.com/JaDogg/pydoro/pull/12  for introducing a config file. However I want to modify it and introduce an `.ini` as a config file so people don't need to edit code to change things. 

## Where to look for config file
1) Look in the location set in `PYDORO_CONFIG_FILE`
2) Look in `~/.pydoro.ini`
3) Look in `current dir/.pydoro.ini`

## Config format
```ini
[keys]
navigate_left = ["left", "s-tab"]

[animation]
no_seconds = False # Only show minutes if this is True.
no_progress = False # Does not animate the indefinite progress bar.

[time]
work_minutes = 25
sbreak_minutes =  5
lbreak_minutes = 15
```

### PR DESCRIPTION
**Pull request by gmelodie**
Adding a configuration file where user can specify key bindings, animation preferences, etc.

Fixes #15

### CODE REVIEW

File: pydoro/pydoro_tui.py [hunk 1]
```diff
@@ -85,6 +89,35 @@ def exit_clicked(_=None):
...
+    with open(homedir + "/.pydoro.ini", "w+") as configfile:
+        config.write(configfile)
+
+
+# Set/Read config file
+config = configparser.ConfigParser()
+
+# Look at PYDORO_CONFIG_FILE environment variable
+# Defaults to ~/.pydoro.ini if PYDORO_CONFIG_FILE not set
+homedir = os.path.expanduser("~")
```
Comment (JaDogg) [inline pydoro/pydoro_tui.py]: Create a separate function to get config.

```
def get_config_file():
    return os.path,abspath(os.path.expanduser("~/.pydoro.ini"))
```

**Comment by gmelodie** [comment]
@JaDogg I didn't figure out how to pass the `work_minutes`, `small_break_minutes` and `long_break_minutes` because it seems they are hard coded into `tomato.py`

**Comment by JaDogg** [comment]
That's a problem. 
Let me think, I'll look at this in the evening and try to come up with a solution.  We'll probably need to do some refactoring to do it 👍

**Comment by gmelodie** [comment]
Maybe we could merge this PR and do the refactoring in another one with #3 (refactoring would make adding keybindings much easier)

**Comment by JaDogg** [comment]
@gmelodie I'll create a `develop` branch. Then point this merge request to `develop` branch. It's easier to do refactoring there and later on merge to master branch.

**Comment by JaDogg** [comment]
@gmelodie Created a develop branch point this pull request to that. (Maybe you need to close this and create a new one)

**Comment by gmelodie** [comment]
@JaDogg Done! So how will we refactor the code? Any ideas?

**Pull request merged**